apps/editing-toolkit/editing-toolkit-plugin/wpcom-global-styles/notices.js

```diff
@@ -114,6 +121,15 @@ function GlobalStylesEditNotice() {
 			},
 		];
 
+		if ( isPostEditor && canPreviewPost ) {
+			actions.push( {
+				label: __( 'Preview without custom styles', 'full-site-editing' ),
```

Comment:
Nitpick: Can we add an external url icon here? It seems that notices also accepts an optional className that:
> className: string (optional) to add custom classes to the button styles.